src/oas3/transformers/securities.ts

```diff
@@ -66,9 +66,13 @@ export function transformToSingleSecurity(
   return undefined;
 }
 
-function transformFlows(flows: DeepPartial<OAuthFlowsObject>): IOauthFlowObjects {
+function transformFlows(flows: Optional<DeepPartial<OAuthFlowsObject>>): IOauthFlowObjects {
   const transformedFlows: IOauthFlowObjects = {};
 
+  if (flows === void 0) {
```

Comment:
Should we also protect against `null` and non-object values too while we're at it?

```ts
    const document: Partial<OpenAPIObject> = {
      components: {
        securitySchemes: {
          t1: { type: 'oauth2', flows: "invalid" },
        },
      },
    };

    const document: Partial<OpenAPIObject> = {
      components: {
        securitySchemes: {
          t1: { type: 'oauth2', flows: null },
        },
      },
    };

    const document: Partial<OpenAPIObject> = {
      components: {
        securitySchemes: {
          t1: { type: 'oauth2', flows: 999 },
        },
      },
    };

    const document: Partial<OpenAPIObject> = {
      components: {
        securitySchemes: {
          t1: { type: 'oauth2', flows: [] },
        },
      },
    };
```

Reply:
Yeah, it could be null, true.